upsetplot/plotting.py

```diff
@@ -104,10 +104,12 @@ def _check_index(df):
     return df
 
 
-def _process_data(df, sort_by, sort_categories_by, subset_size, sum_over):
+def _process_data(df, sort_by, sort_categories_by, subset_size,
+                  sum_over, min_subset_size=1):
     df, agg = _aggregate_data(df, subset_size, sum_over)
     df = _check_index(df)
-
+    agg = agg[agg >= min_subset_size]
```

Comment:
Surely you need to do this filtering after `totals` has been calculated?

Reply:
`plot_totals` function uses `totals` to plot barplot. If `totals` contains subsets that are already filtered, it will plot those groups too. We can either filter before calculating `totals` (therefore `totals` only has retained groups) or introduce a new `sub_totals` attribute and use it in `plot_totals`

Reply:
Yes, I think the totals should include the filtered-out subsets.